.docker/docker_run_git.sh

```diff
@@ -113,6 +113,27 @@ if [ $PS_DEMO_MODE -ne 0 ]; then
     sed -i -e "s/define('_PS_MODE_DEMO_', false);/define('_PS_MODE_DEMO_',\ true);/g" /var/www/html/config/defines.inc.php
 fi
 
-echo "\n* Almost ! Starting web server now\n";
+if [ $PS_USE_DOCKER_MAILDEV -eq 1 ]; then
+    echo "\n* Configuring emails to use maildev ..."
+    runuser -g www-data -u www-data -- php /var/www/html/bin/console prestashop:config set PS_MAIL_METHOD --value "2"
+    runuser -g www-data -u www-data -- php /var/www/html/bin/console prestashop:config set PS_MAIL_SERVER --value "maildev"
+    runuser -g www-data -u www-data -- php /var/www/html/bin/console prestashop:config set PS_MAIL_SMTP_PORT --value "1025"
+fi
+
+echo "\n***"
+echo "**"
+echo "** To view storefront point your browser to http://localhost:8001/"
+echo "** To view backoffice point your browser to http://localhost:8001/admin-dev"
+echo "**   Login with:"
+echo "**     username: ${ADMIN_MAIL}"
+echo "**     password: ${ADMIN_PASSWD}"
```

Comment:
I'm not a fan of dispaying the password.

Reply:
This is for development and getting new developer up and running fast it is probably good to give the necessary information to them.

But I do get the point, maybe the password could say something like "the password you defined or "Correct Horse Battery Staple" if you didn't specify your own"?

Reply:
I agree with @nicosomb on this one.
@tswfi, I personally prefer your second proposal! :D

Reply:
Changed to use the default password and comment "if you didn't define your own" https://github.com/PrestaShop/PrestaShop/pull/32208/commits/0f93493dbc33b4d9d1774001dcbff188d0628e04